frontend/public/components/resource-list.tsx

```diff
@@ -99,18 +99,20 @@ export const ResourceDetailsPage = connectToPlural((props: ResourceDetailsPagePr
     props.match.path.indexOf('customresourcedefinitions') === -1
       ? referenceForModel(kindObj)
       : null;
-  const componentLoader = getResourceDetailsPages(
-    detailsPageExtensions,
-    dynamicResourceListPageExtensions,
-  ).get(ref, () => Promise.resolve(DefaultDetailsPage));
+  const componentLoader =
+    getResourceDetailsPages(detailsPageExtensions, dynamicResourceListPageExtensions).get(ref) ||
+    getResourceDetailsPages(detailsPageExtensions, dynamicResourceListPageExtensions).get(
+      `${kindObj.apiGroup}~~${kindObj.kind}`,
+    );
```

Comment:
I'd prefer that you construct an object with `apiGroup` and `kind` and use one of the `referenceFor*` functions instead of constructing the reference here like this. By doing this, the details on how to construct a reference is owned by a single code location.